packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -212,6 +213,7 @@ export interface ProcessDetail {
 
 // From backend
 export enum ProcessStatus {
+    CREATE = 'create', // Frontend only
```

Comment:
"From backend" and "Frontend only" is a bit confusing. Think it might be better to have a more strict split between types from backend and what we use in frontend. (low prio for now in my opinion)

Reply:
Added some more explanation